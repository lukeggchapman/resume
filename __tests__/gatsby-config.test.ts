const gatsbyConfig = require('../gatsby-config')

describe('Gatsby Config', () => {
  it('config has not changed', () => {
    global.__dirname = 'some/path' // eslint-disable-line

    expect(gatsbyConfig).toMatchSnapshot()
  })
})
