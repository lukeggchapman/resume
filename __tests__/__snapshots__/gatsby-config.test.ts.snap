// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gatsby Config config has not changed 1`] = `
Object {
  "plugins": Array [
    "gatsby-plugin-react-helmet",
    Object {
      "options": Object {
        "name": "images",
        "path": "src/images",
      },
      "resolve": "gatsby-source-filesystem",
    },
    Object {
      "options": Object {
        "name": "education",
        "path": "content/education",
      },
      "resolve": "gatsby-source-filesystem",
    },
    Object {
      "options": Object {
        "name": "experience",
        "path": "content/experience",
      },
      "resolve": "gatsby-source-filesystem",
    },
    Object {
      "options": Object {
        "name": "skills",
        "path": "content/skills",
      },
      "resolve": "gatsby-source-filesystem",
    },
    "gatsby-transformer-remark",
    "gatsby-transformer-sharp",
    "gatsby-transformer-json",
    "gatsby-plugin-sharp",
    "gatsby-plugin-emotion",
    "gatsby-plugin-eslint",
    Object {
      "options": Object {
        "display": "minimal-ui",
        "icon": "src/images/chappo-icon.png",
        "name": "luke-chapman-resume",
        "short_name": "resume",
        "start_url": "/",
      },
      "resolve": "gatsby-plugin-manifest",
    },
    Object {
      "options": Object {
        "pathToConfigModule": "src/utils/typography",
      },
      "resolve": "gatsby-plugin-typography",
    },
  ],
  "siteMetadata": Object {
    "author": "lukeggchapman",
    "description": "Résumé PDF generated using Gatsby and React-PDF",
    "title": "Luke Chapman Résumé",
  },
}
`;
